### PR TITLE
updated es_ES locale

### PR DESCRIPTION
**Pull request by mtub**
* files submitted by @marcbria

#1232

### CODE REVIEW

File: locale/es_ES/manager.xml [hunk 1]
```diff
@@ -135,91 +162,291 @@
...
+	<p>También puede elegir qué módulo analizador ParaTools se debe usar. You can choose between the standard, <a href="http://search.cpan.org/~mjewell/Biblio-Citation-Parser-1.10/lib/Biblio/Citation/Parser/Jiao.pm" target="_blank">Jiao</a>, and <a href="http://search.cpan.org/~mjewell/Biblio-Citation-Parser-1.10/lib/Biblio/Citation/Parser/Citebase.pm" target="_blank">Citebase</a> parser modules.</p>]]></message>
```
Comment (asmecher) [inline locale/es_ES/manager.xml]: This appears to swap the Spanish out for English.

File: locale/es_ES/reader.xml [hunk 2]
```diff
@@ -4,19 +4,19 @@
 <!--
   * locale/es_ES/reader.xml
   *
-  * Copyright (c) 2014-2015 Simon Fraser University Library
-  * Copyright (c) 2000-2015 John Willinsky
+  * Copyright (c) 2014 Simon Fraser University Library
+  * Copyright (c) 2000-2014 John Willinsky
```
Comment (asmecher) [inline locale/es_ES/reader.xml]: Copyright dates reverted accidentally

File: locale/es_ES/reader.xml [hunk 3]
```diff
@@ -4,19 +4,19 @@
...
-  *
+  * Credits: https://pkp.sfu.ca/wiki/index.php?title=OMP:_Spanish_(es_ES)
+  * Localization strings.
   -->
 
-<locale name="es_ES" full_name = "Español (España)">
+<locale name="es_ES" full_name="Español (España)">
 	<message key="comments.anonymous">Usuario anónimo</message>
-	<message key="comments.anonymousNamed">"{$userName}"</message>
+	<message key="comments.anonymousNamed">&quot;{$userName}&quot;</message>
```
Comment (asmecher) [inline locale/es_ES/reader.xml]: The quoting before was consistent with the en_US translation; not sure why quot entities are used here.

File: locale/es_ES/reader.xml [hunk 4]
```diff
@@ -40,18 +40,37 @@
 	<message key="rt.admin.contexts.createContext">Crear contexto</message>
 	<message key="rt.admin.contexts.edit.editContext">Editar contexto</message>
 	<message key="rt.admin.contexts.metadata">Metadatos</message>
-	<message key="rt.admin.contexts.options.authorTerms">Usar nombres de autores como términos de búsqueda por defecto para permitir a los usuarios encontrar otros trabajos de los autores de este artículo (p.ej., para un contexto de "Otros trabajos")</message>
+	<message key="rt.admin.contexts.options.authorTerms">Usar nombres de autores como términos de búsqueda por defecto para permitir a los usuarios encontrar otros trabajos de los autores de este artículo (p.ej., para un contexto de &quot;Otros trabajos&quot;)</message>
```
Comment (asmecher) [inline locale/es_ES/reader.xml]: Likewise, I'm nervous here about using the `quot` entity where `"` is perfectly fine. In some cases it'll probably cause the `&quot;` to be displayed verbatim in the output and/or emails where it's included.

File: locale/es_ES/user.xml [hunk 5]
```diff
@@ -4,25 +4,36 @@
 <!--
   * locale/es_ES/user.xml
   *
-  * Copyright (c) 2014-2015 Simon Fraser University Library
-  * Copyright (c) 2000-2015 John Willinsky
+  * Copyright (c) 2014 Simon Fraser University Library
+  * Copyright (c) 2000-2014 John Willinsky
```
Comment (asmecher) [inline locale/es_ES/user.xml]: As elsewhere, copyright dates

File: locale/es_ES/reader.xml (hunk 4 above)
Comment (mtub) [inline locale/es_ES/reader.xml]: Sorry, I thought I caught all of those. Let me have another look…

**Comment by mtub** [comment]
Should be okay now, all issues fixed.

File: locale/es_ES/manager.xml [hunk 6]
```diff
@@ -116,7 +143,7 @@
 	<message key="manager.people.enrollUserAs">Inscribir usuaria/o como</message>
 	<message key="manager.people.invalidUser">Lo sentimos, la/el usuaria/o solicitada/o no existe.</message>
 	<message key="manager.people.mergeUser">Unir usuario/a</message>
-	<message key="manager.people.mergeUsers.confirm">¿Seguro que quiere unir la cuenta "{$oldAccountCount}" con la cuenta "{$newUsername}"? La cuenta "{$oldAccountCount}" se eliminará.  Esta acción no es reversible.</message>
+	<message key="manager.people.mergeUsers.confirm">Are you sure you wish to merge the selected {$oldAccountCount} account(s) into the account with the username "{$newUsername}"? The selected {$oldAccountCount} accounts will not exist afterwards. Esta acción es irreversible.</message>
```
Comment (asmecher) [inline locale/es_ES/manager.xml]: Some English crept in here.

Comment (marcbria) [inline locale/es_ES/manager.xml]: This should be:

> ¿Está seguro que desea unir la cuenta "{$oldAccountCount}" con la cuenta "{$newUsername}"? La cuenta "{$oldAccountCount}" se eliminará. Esta acción es irreversible."

**Comment by mtub** [comment]
Should be okay now, sorry for missing these lines before.

**Comment by asmecher** [comment]
Merged, thanks!

**Pull request merged**